docs/docs/architecture/attestation.md

```diff
@@ -40,7 +40,8 @@ In the userland, the guest agent takes care of enforcing the [runtime policy](..
 While the policy is passed in during the initialization procedure via the host, the evidence for the runtime policy is part of the CPU measurements.
 During the [deployment](../deployment.md#generate-policy-annotations-and-manifest) the policy is annotated to the Kubernetes Pod resources.
 On AMD SEV-SNP the hash of the policy is then added to the attestation report via the `HOSTDATA` field by the hypervisor.
-When provided with the policy from the Kata host, the guest agent verifies that the policy's hash matches the one in the `HOSTDATA` field.
+On Intel TDX the hash of the policy is then added to the attestation report via the `MRCONFIGID` field by the hypervisor.
```

Comment:
The duplication reads a bit weird - maybe a table, bullet points or something like that would help?

Reply:
How about 
> The hypervisor adds the hash of the policy to the attestation report via the `HOSTDATA` (on AMD SEV-SNP) or `MRCONFIGID` (Intel TDX) fields.